docs/reference/Signed-Package-Verification-Options.md

```diff
@@ -24,10 +24,13 @@ For NuGet users, symptoms of this issue are that the NuGet operation will typica
 > [!Note]
 > This option is available starting from NuGet 6.0.0 and only applies to the Windows-specific failure described above.  The option does not apply to any other scenario and has no effect on Linux or macOS.
 
-You can opt-in to an experimental, automatic retry for untrusted root failures on Windows by setting an environment variable named `NUGET_EXPERIMENTAL_CHAIN_BUILD_RETRY_POLICY` with a value consisting of 2 comma-delimited positive integers representing retry count and sleep interval in milliseconds, respectively. There are no default values; you need to pick retry values that are sensible for you.
+You can opt in to an experimental, automatic retry for untrusted root failures on Windows by setting an environment variable named `NUGET_EXPERIMENTAL_CHAIN_BUILD_RETRY_POLICY` with a value consisting of 2 comma-delimited positive integers representing retry count and sleep interval in milliseconds, respectively. There are no default values; you need to pick retry values that are sensible for you.
 
 For example, setting the environment variable to a value of `3,1000` like so:
 
 <pre>set NUGET_EXPERIMENTAL_CHAIN_BUILD_RETRY_POLICY=3,1000</pre>
 
 ...would try up to 4 times (initial try plus 3 retries) with 1 second (1,000 ms) between each try.
+
+> [!Note]
```

Comment:
Should this note be combined with the note above? 
That way a new reader sees that they don't need to set it starting with 8.0.100.